src/test/Global.cpp

```diff
@@ -20,12 +20,12 @@
 
 #include <sstream>
 
-#include "ILogger.h"
-#include "Singleton.tpp"
+#include "utility/ILogger.h"
+#include "utility/Singleton.tpp"
```

Comment:
```suggestion
#include "../utility/Singleton.tpp"
```

(Same for `ILogger`)

Otherwise, including this header in a third-party project results in a compiler error as `include/DrMock` is not an include path (nor should it!).

By the way, this is another argument for linking tests against a temporary installation: This kind of error can only be caught by linking the tests against a proper installation (see https://github.com/DrCpp/DrMock/pull/1#discussion_r549804420).

Reply:
Fixed here and everywhere else.